libnavigation-metrics/src/main/java/com/mapbox/navigation/metrics/MapboxMetricsReporter.kt

```diff
@@ -98,4 +109,34 @@ object MapboxMetricsReporter : MetricsReporter {
     override fun removeObserver() {
         this.metricsObserver = null
     }
+
+    private class LocationSerializer : JsonSerializer<Location> {
+        override fun serialize(
+            location: Location,
+            type: Type,
+            context: JsonSerializationContext
+        ): JsonElement {
+            return JsonObject().apply {
+                addProperty(LATITUDE, location.latitude)
+                addProperty(LONGITUDE, location.longitude)
+                addProperty(SPEED, location.speed)
+                addProperty(COURSE, location.bearing)
+                addProperty(ALTITUDE, location.altitude)
+                addProperty(TIMESTAMP, location.time)
```

Comment:
Should the timestamp be in seconds or millis?

Reply:
I guess it's expected a default `Location` timestamp, so in milliseconds.
@shvrm @NightBrownie cc